src/Projects/Server/Cida.Server/Module/CidaModule.cs

```diff
@@ -62,10 +62,10 @@ private CidaModuleLoadContext InitializeLoadContext()
             return context;
         }
 
-        public async Task<IModule> Load(IDatabaseConnector databaseConnector)
+        public async Task<IModule> Load(IDatabaseConnector databaseConnector, Cida.Api.IFtpClient ftpClient)
```

Comment:
is there a reason for fully qualifying `IFtpClient` here?

Reply:
Not a profound one. Will go through the PR and remove unnecessary full qualifies